### ISSUE TITLE
feat(a11y): Add aria-label and add them to translation.json-files

### ISSUE THREAD

**Issue by cguhl-adesso** [Rudo-hayate/open-webui-a11y#9]
### Check Existing Issues

- [x] I have searched for all existing **open AND closed** issues and discussions for similar requests. I have found none that is comparable to my request.

### Verify Feature Scope

- [x] I have read through and understood the scope definition for feature requests in the Issues section. I believe my feature request meets the definition and belongs in the Issues section instead of the Discussions.

### Problem Description

There are a few aria-label missing to support a11y. Additionally, some aria-label are hardcoded and should be extracted to the language files.
The vite-plugin-svelte also shows them es warning when starting frontend.

### Desired Solution you'd like

- add aria-label
- use aria-label from language-files (german, english)
